docs/src/developing/programming-model/runtime.md

```diff
@@ -131,14 +135,22 @@ Budget](#compute-budget).
 The transaction-wide compute budget applies the `max_units` cap to the entire
 transaction rather than to each instruction within the transaction. The default
 transaction-wide `max_units` will be calculated as the product of the number of
-instructions in the transaction by the default per-instruction units, which is
-currently 200k. During processing, the sum of the compute units used by each
-instruction in the transaction must not exceed that value. This default value
-attempts to retain existing behavior to avoid breaking clients. Transactions can
-request a specific number of `max_units` via [Compute Budget](#compute-budget)
-instructions.  Clients should request only what they need; requesting the
-minimum amount of units required to process the transaction will improve their
-fee/compute-unit ratio, which transaction prioritization is based on.
+instructions in the transaction (excluding [Compute Budget](#compute-budget)
+instructions) by the default per-instruction units, which is currently 200k.
+During processing, the sum of the compute units used by each instruction in the
+transaction must not exceed that value. This default value attempts to retain
+existing behavior to avoid breaking clients. Transactions can request a specific
+number of `max_units` via [Compute Budget](#compute-budget) instructions.
+Clients should request only what they need; requesting the minimum amount of
+units required to process the transaction will reduce overall transaction cost,
+which may include a prioritization-fee charged for every compute unit.
+
+Transaction prioritization is determined by the transactions prioritization fee
+which itself is the product of the transaction's compute unit budget and its
+compute unit price (measured in micro-lamports). The compute unit budget and
```

Comment:
still reading through this the code piece of this, but i was under assumption you'd want to get prioritized on unit of time? 

would you rather get paid $60 for 1 minute of work ($3600/hr) or $10 for 361 hours? from this description it seems like the algorithm would choose the $10 for 361 hours

Reply:
Ah yeah, just a doc mistake. We're prioritizing on the rate now. Thanks for pointing out

Reply:
sounds good haha

Reply:
PR here: https://github.com/solana-labs/solana/pull/25239